test/tools/IsolTestOptions.cpp

```diff
@@ -56,14 +56,21 @@ std::string editorPath()
 
 }
 
-IsolTestOptions::IsolTestOptions(std::string* _editor):
+IsolTestOptions::IsolTestOptions():
 	CommonOptions(description)
 {
+	enforceViaYul = true;
+	enforceGasTest = (evmVersion() == langutil::EVMVersion{});
+}
```

Comment:
What happened to `enforceGasTestMinValue = 100000;`?

Reply:
It is the same default value as in the base class, @cameel wanted to remove the redundancy.

Reply:
Yeah, it was redundant. Hard-coding it in two places does not seem like a good idea and changing it to `enforceGasTestMinValue = enforceGasTestMinValue` would be pointless.

Reply:
Ok it was [here](https://github.com/ethereum/solidity/pull/11996#discussion_r712135031), I just didn't expand all the closed review comments.